sdcm/cluster_k8s/__init__.py

```diff
@@ -153,7 +153,9 @@ def __init__(self, name: str, parent_cluster: "PodCluster", node_prefix: str = "
 
     def init(self) -> None:
         super().init()
-        self.remoter.run('mkdir -p /var/lib/scylla/coredumps', ignore_status=True)
+        if self.distro.is_rhel_like:
+            self.remoter.sudo("rpm -q iproute || yum install -y iproute")  # need this because of scylladb/scylla#7560
+        self.remoter.sudo('mkdir -p /var/lib/scylla/coredumps', ignore_status=True)
```

Comment:
how is the issue related to coredumps directory?

Reply:
It was here before my changes. It's added by Dmitry.